frontend/src/Utils/api-discovery.types.ts

```diff
@@ -0,0 +1,79 @@
+import { K8sModelCommon } from '@openshift/dynamic-plugin-sdk-utils';
+import { Action, Store } from 'redux';
+
+// eslint-disable-next-line no-shadow
+export declare enum ActionType {
+  ReceivedResources = 'resources',
+  SetResourcesInFlight = 'setResourcesInFlight',
+  SetBatchesInFlight = 'setBatchesInFlight',
+  StartWatchK8sObject = 'startWatchK8sObject',
+  StartWatchK8sList = 'startWatchK8sList',
+  ModifyObject = 'modifyObject',
+  StopWatchK8s = 'stopWatchK8s',
+
+  Errored = 'errored',
+  Loaded = 'loaded',
+  BulkAddToList = 'bulkAddToList',
+  UpdateListFromWS = 'updateListFromWS',
+  FilterList = 'filterList',
+}
+
+export type K8sResourceKindReference = string;
```

Comment:
@karelhala this is deprecated in the sdk. Can we indicate that here too?

Edit: actually this looks to be explicitly exported too: https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-utils/src/index.ts#L38

Reply:
Yup, these are the type issues I was facing. Not sure why I had to move them here, but tslint was complaining about not being accessible.

Reply:
@karelhala I debugged this locally. @vojtechszocs's change to use the API extractor tool fixed this issue (missing type exports). 

I will publish lib-utils@1.0.1 tomorrow to resolve this issue.